bouncer/util.py

```diff
@@ -70,4 +101,35 @@ def parse_document(document):
             _("The annotation has an invalid document URI"),
             "uri_not_a_string")
 
-    return (annotation_id, document_uri)
+    return {
+            "annotation_id": annotation_id,
+            "document_uri": document_uri,
+            "can_reveal_metadata": can_reveal_metadata,
+            "quote": quote,
+            "text": text
+            }
+
+
+def make_pretty_url(url):
+    """
+    Return the domain name from `url` for display.
+    """
+    try:
+        parsed_url = parse.urlparse(url)
+        pretty_url = parsed_url.netloc[:NETLOC_MAX_LENGTH]
+        if len(parsed_url.netloc) > NETLOC_MAX_LENGTH:
+            pretty_url = pretty_url + jinja2.Markup("&hellip;")
+    except:
+        return "CannotParseURL"
+    return pretty_url
+
+
+def make_boilerplate_quote(document_uri):
+    if document_uri is None:
+        return "Hypothesis annotation"
+    else:
+        pretty_url = make_pretty_url(document_uri)
+        return "Hypothesis annotation for {site}".format(site=pretty_url)
+
+def make_boilerplate_text():
```

Comment:
There's no real reason for this to be a function, it would be much nicer and easier to follow if this is a constant at the top of the file. Also the string should be translatable, you need to wrap it into a `_()` function call for that (e.g. `_('Follow this link ...')`)